javascript/net/grpc/web/grpc_generator.cc

```diff
@@ -723,6 +726,7 @@ void PrintProtoDtsMessage(Printer *printer, const Descriptor *desc, const FileDe
 
 void PrintProtoDtsFile(Printer *printer, const FileDescriptor *file)
 {
+  printer->Print("import * as jspb from \"google-protobuf\"\n\n");
```

Comment:
Test failure:
```
/github/grpc-web/net/grpc/gateway/examples/echo/ts-example/node_modules/grpc-web -> /usr/local/lib/node_modules/grpc-web -> /github/grpc-web/packages/grpc-web
echo_pb.d.ts(1,23): error TS7016: Could not find a declaration file for module 'google-protobuf'. '/github/grpc-web/net/grpc/gateway/examples/echo/ts-example/node_modules/google-protobuf/google-protobuf.js' implicitly has an 'any' type.
  Try `npm install @types/google-protobuf` if it exists or add a new declaration (.d.ts) file containing `declare module 'google-protobuf';`
Service 'ts-client' failed to build: The command '/bin/sh -c cd $EXAMPLE_DIR/ts-example &&   npm install &&   npm link grpc-web &&   tsc &&   npx webpack &&   cp echotest.html /var/www/html &&   cp dist/main.js /var/www/html/dist' returned a non-zero code: 2
```

The problem with this is that, the `google-protobuf` package does not publish a `.d.ts` typings.

Reply:
adding `@types/google-protobuf` to `package.json` might help?